src/mscorlib/src/System/Reflection/AssemblyName.cs

```diff
@@ -533,7 +571,83 @@ internal string GetNameWithPublicKey()
         private extern byte[] nGetPublicKeyToken();
     
         [System.Security.SecurityCritical]  // auto-generated
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        static internal extern String EscapeCodeBase(String codeBase);
+        static internal String EscapeCodeBase(String codebase)
+        {
+            return EscapeString(codebase, false, true, true);
+        }
+
+        static internal String EscapeString(String str, bool escapeReserved, bool escapeHex, bool escapeBrackets)
```

Comment:
It is just a few methods - I would create copy of the corefx impl in in CoreLib.